jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -49,11 +49,13 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
+import lombok.ToString;
 
+@ToString(exclude = {"registry", "clock", "metrics"})
 public class Tracer implements io.opentracing.Tracer {
-  public static final String VERSION = loadVersion();
```

Comment:
This information depends on the runtime class path, and I didn't feel that it's sufficiently different from other things that depend on initialization like the `serviceName` and `ip`. I could change it back if you feel strongly about it.